Adding complex password for pfx as a requirement.

sameeraman opened the pull request:
Today during an Azure Stack deployment we found out that the pfx certificates should have a complex password. Otherwise, deployment fails.

azurecla commented:
Hi __@singhkay__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (kasing). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

tysonn commented:
@singhkay - Please see my updates pending in a pull request against your fork. That needs to be accepted prior to this.

singhkay commented:
@tysonn merged your PR

sameeraman committed:
Adding complex password for pfx as a requirement.

Today during an Azure Stack deployment we found out that the pfx certificates should have a complex password. Otherwise, deployment fails.
